man/git-reauthor.md

```diff
@@ -0,0 +1,83 @@
+git-reauthor(1) -- Rewrite history to change author's identity
+==============================================================
+
+## SYNOPSIS
+
+`git reauthor [<options>]`
+
+## DESCRIPTION
+
+Rewrite history to change author's identity.
```

Comment:
This line is the same as next to the `Name` field. Should be removed.

We have a tradition of sorts of repeating the short description (the one next to the name) in the description, but I have never seen anyone else do that. I think we should stop doing it.